src/lib/rest/RestService.cpp

```diff
@@ -138,7 +140,14 @@ static std::string tenantCheck(const std::string& tenant)
 
   if (strlen(name) > SERVICE_NAME_MAX_LEN)
   {
-    LM_W(("Bad Input (a tenant name can be max %d characters long. Length: %d)", SERVICE_NAME_MAX_LEN, strlen(name)));
+    char numV1[16];
```

Comment:
Then, let's define a LENGTH_ENOUGHT_TO_HOLD_AND_INTEGER (or course, not that literal name ;) and use it in every place it makes sense. That way if we discover that integers are larger than we expected we can change *in a single place* and benefit everywhere.

Makes sense?